### PR TITLE
Move some assertion tests to skipped e2e tests block

### PR DESCRIPTION
**Pull request by Myrfion**
## Please check the following

- [x] Do the tests still pass? (see [Run the Tests](https://github.com/cuttle-cards/cuttle#run-the-tests))
- [x] Is the code formatted properly? (see [Linting (Formatting)](https://github.com/cuttle-cards/cuttle#linting-formatting))
- For New Features:
  - [x] Have tests been added to cover any new features or fixes?
  - [x] Has the documentation been updated accordingly?

## Please describe additional details for testing this change
Changes include refactoring existing E2E tests and moving some 'Aesthetic tests', which are not necessarily have to be runned on the CI to `describe.skip('Aesthetic tests'` block to increase the CI performance 

Issue #192

### CODE REVIEW

File: tests/e2e/specs/in-game/layout.spec.js [hunk 1]
```diff
@@ -270,206 +474,4 @@ describe('Game View Layout', () => {
...
-      p0Points: [],
-      p0FaceCards: [],
-      p1Hand: [Card.TEN_OF_HEARTS],
-      p1Points: [],
-      p1FaceCards: [],
-      scrap: [Card.FOUR_OF_CLUBS, Card.FOUR_OF_SPADES, Card.ACE_OF_DIAMONDS],
-    });
-  });
-});
+})
```
Comment (itsalaidbacklife) [inline tests/e2e/specs/in-game/layout.spec.js:477]:
```suggestion
});

```

**Review by itsalaidbacklife — COMMENTED** [review]
Looks great, thank you! Could you please bump the version field in package.json to the next patch? That should be 3.2.10

**Comment by Myrfion** [comment]
@itsalaidbacklife done 👍

**Review by itsalaidbacklife — APPROVED** [review]
Thanks for your contribution! Please feel free to to take on another issue if you'd like. I'd also be happy to help you find one that suits your interests and/or learning goals.

Regardless, thanks for contributing to cuttle.cards!

**Pull request merged**